todos/app/resources/lists/resource.rb

```diff
@@ -20,8 +20,8 @@ class Resource < ApplicationResource
     end
 
     extend_model do
-      def self.sorting(order_by, order)
-        sort = parse_sorting_names(order_by) || 'created_at'
+      def self.sorting(order_by = 'created_at', order = 'asc')
+        sort = parse_sorting_names(order_by)
         joins(:project).order(sort => order || 'asc')
```

Comment:
The current implementation requires both a default value for the argument AND a default value here, because when `params[order_by].present? && !params[order]present?` the method would be called with (order_by, nil) and would break line 25.